### ISSUE TITLE
[BUG] "Unknown type" in Language injection

### ISSUE THREAD

**Issue by MegaJSDeveloper** [jimkyndemeyer/js-graphql-intellij-plugin#307]
![image](https://user-images.githubusercontent.com/58376230/71845659-d81c6f00-30d9-11ea-900d-6405215059b1.png)
version 2.3.0

**Comment by jimkyndemeyer** [comment]
Hi,

Thanks for using the plugin.

Schema discovery looks for tagged template literals named `gql` or `graphql`. This is done for performance reasons to support very large projects.

If you prefer to declare your schema types in a JS file, and don't use frameworks such as Apollo or Relay, you can define your own simple graphql function, in which types are picked up by the plugin, as follows:

```typescript
function graphql(strings, ...placeholders) {
    return strings[0];
}

const schema = graphql`
    type User {
        name: String
    }
    
    type Query {
        user: User
    }
`;
```

Best regards,
Jim.

**Comment by MegaJSDeveloper** [comment]
@jimkyndemeyer yes, but I just discovered a bug. you did not know, but language = LANGNAME is the standard feature of JetBrains to activate Language Injection

**Comment by vepanimas** [comment]
Related #308
